docs/source/recipes/recipes_overview.rst

```diff
@@ -0,0 +1,68 @@
+.. _recipes_overview_label:
+
+================
+Recipes Overview
+================
+
+Recipes are the primary entry points for torchtune users.
+These can be thought of as **hackable, singularly-focused scripts for interacting with LLMs** including fine-tuning,
+inference, evaluation, and quantization.
+
+Each recipe consists of three components:
+
+* **Configurable parameters**, specified through yaml configs and command-line overrides
+* **Recipe script**, entry-point which puts everything together including parsing and validating configs, setting up the environment, and correctly using the recipe class
+* **Recipe class**, core logic needed for fine-tuning, exposed through a set of APIs
+
+.. note::
+
+  To learn more about the concept of "recipes", check out our technical deep-dive: :ref:`recipe_deepdive`.
+
+.. note::
+
+    You can also utilize the :code:`tune ls` command to print out all recipes and corresponding configs.
+
+    .. code-block:: bash
+
+        $ tune ls
+        RECIPE                                   CONFIG
+        full_finetune_single_device              llama2/7B_full_low_memory
+                                                mistral/7B_full_low_memory
+        full_finetune_distributed                llama2/7B_full
+                                                llama2/13B_full
+                                                mistral/7B_full
+        lora_finetune_single_device              llama2/7B_lora_single_device
+                                                llama2/7B_qlora_single_device
+                                                mistral/7B_lora_single_device
+        ...
+
+Supervised finetuning
+---------------------
+
+Torchtune provides built-in recipes for finetuning on single device, on multiple devices with `FSDP <https://pytorch.org/blog/introducing-pytorch-fully-sharded-data-parallel-api/>`_,
+using a variety :ref:`of memory optimization features <memory_optimisation_overview_label>`. Our  fine-tuning recipes support all of our models and all our dataset types.
+This includes continued pre-training, and various supervised fune-tuning paradigms, which can be customised through our datasets. Check out our
+:ref:`dataset tutorial <dataset_tutorial_label>` for more information. Our supervised fine-tuning recipes include:
+
+* :ref:`Single-device <lora_finetune_recipe_label>` LoRA fine-tuning.
```

Comment:
do you plan to add all the other recipes to this list?

Reply:
I'd like to file some issues to lean on community contributions for some of the other recipes : ) I'll add DPO, and PPO when it's landed for sure.